packages/react-components/src/types/ComponentSlotStyle.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IRawStyle } from '@fluentui/react';
+
+/**
```

Comment:
Would this be better to put this one in northstar wrapper?

Reply:
I did consider that but I'm actually against that as we shouldn't ever have publicly exported a fluent northstar interface and should have defined our own interface/used regular fluents in this package.

Reply:
(It also makes the bundling more difficult if we have a public API coming out of the northstar wrapper the way we rollup the api)